src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class BalanceOperation implements OperationHandler {
+
+    @Override
+    public void handle(Storage storage, FruitTransaction transaction) {
+        storage.setFruitQuantity(transaction.getFruit(), transaction.getQuantity());
```

Comment:
The `handle` method directly manipulates the `storage` object by setting the fruit quantity. This could lead to problems in the future if the `storage` object changes. A better approach might be to have the `storage` object handle its own state changes. For example, you could add a method to the `storage` class like `updateFruitQuantity` and use that method here instead.

Reply:
It doesn't lead to problems